src/navigation/drive.py

```diff
@@ -44,8 +46,68 @@ def get_drive_command(
     if target_dist == 0:
         target_dist = np.finfo(float).eps
 
+
     alignment = angle_to_rotate(rover_dir, target_dir)
+    if target_dist < completion_thresh:
+        return Twist(), True
+
+    cmd_vel = Twist()
+    full_turn_override = True
+    if abs(alignment) < turn_in_place_thresh:
+        error = target_dist
+        cmd_vel.linear.x = np.clip(error, 0.0, MAX_DRIVING_EFFORT)
+        full_turn_override = False
+
+    # we want to drive the angular offset to zero so the error is just 0 - alignment
+    error = alignment
+
+    cmd_vel.angular.z = (
+        np.sign(error) if full_turn_override else np.clip(error * TURNING_P, MIN_DRIVING_EFFORT, MAX_DRIVING_EFFORT)
+    )
+
+    print(cmd_vel.linear.x, cmd_vel.angular.z)
+    return cmd_vel, False
+
+
+def get_j_turn_command(
```

Comment:
so this should probably just be named get_reverse_drive_command since that's what it really is. Also we should try to take as much functionality in common between this and the get_drive_command function and put it in a separate function to reduce duplication